sdk/typescript/src/types/transactions.ts

```diff
@@ -65,17 +65,19 @@ export const SuiArgument = union([
   object({ Result: number() }),
   object({ NestedResult: tuple([number(), number()]) }),
 ]);
+export type SuiArgument = Infer<typeof SuiArgument>;
+
+const MoveCallSuiCommand = object({
```

Comment:
If you're exporting the type, also export the struct.

Reply:
added export